renderer-macos/lib/stubs/BriskStylableText.c

```diff
@@ -0,0 +1,107 @@
+#import "BriskStylableText.h"
+
+void ml_BriskStylableText_applyChanges(id<BriskStylableText> txt) {
+  [txt applyTextStyle];
+}
+
+CAMLprim value ml_BriskStylableText_setFont(id<BriskStylableText> txt,
+                                            value fontName_v, double fontSize,
+                                            double fontWeight) {
+  CAMLparam1(fontName_v);
+
+  NSString *fontName = [NSString stringWithUTF8String:String_val(fontName_v)];
+  NSFont *font;
+
+  if (fontName.length > 0) {
+    NSDictionary *attributes = @{
+      NSFontFaceAttribute : fontName,
+      NSFontTraitsAttribute :
+          @{NSFontWeightTrait : [NSNumber numberWithDouble:fontWeight]}
+    };
+
+    NSFontDescriptor *descriptor =
+        [NSFontDescriptor fontDescriptorWithFontAttributes:attributes];
+
+    font = [NSFont fontWithDescriptor:descriptor size:(CGFloat)fontSize];
```

Comment:
IIRC, Cocoa will just fallback to default font.

Reply:
We might want to throw an exception, because not knowing whether it's your font that is wrong or it's simply not being able to load is bad DX.

But then again: there has to be a way to manage it.

Reply:
indeed!